ls1intum/Artemis
# Fix several small CSS issues related to exam conduction

### Checklist

### Motivation and Context

Closes #3590.

### Description

Adjusted the buttons for adding exercises to exercise groups overflowing outside of the page. Now the text is replaced by the respective icons on smaller screens:
![grafik](https://user-images.githubusercontent.com/72132281/122584336-87dd6680-d05a-11eb-86a8-5faddce8b47f.png)

I'd suggest we add a unique icon for "Import" in a follow-up.



The width of the warnings on the exam/test run start cover now have less width:
![grafik](https://user-images.githubusercontent.com/72132281/122580578-64182180-d056-11eb-8ce9-11af2df88802.png)

And finally the text of the exercise dashboard button does no longer disappear on smaller screen sizes:
![grafik](https://user-images.githubusercontent.com/72132281/122580626-71351080-d056-11eb-817c-8501e68de075.png)

However, none of these pages are meant to be viewed at such small sizes or rather, written for it. Example (when you submit an assessment):
![grafik](https://user-images.githubusercontent.com/72132281/122580719-8c078500-d056-11eb-9b89-23fab0681863.png)

So while I fixed those issues I don't think we should change much more here without a plan on how to adjust the entire exam mode (or parts of it) for smaller screen sizes.

### Steps for Testing

Check the screenshots in the issue and compare them with the screenshots here. Check if the pages look the same as on my screenshots on the test servers/locally.
